leases/models.py

```diff
@@ -53,7 +54,7 @@ class Meta:
 
     @property
     def order(self):
-        return self._orders_relation.first()
+        return self._orders_relation.filter(order_type=OrderType.LEASE_ORDER).first()
```

Comment:
Not sure if it would make sense to keep this prop now that leases can have multiple orders. Let's rename the `_orders_relation` field to just `orders` so we can do `lease.orders.first()` if we need this.

(In any case, we're not using it anywhere.)

Reply:
The FE is using the `order` when displaying the invoice on application page.